device-types/Palo Alto/PA-440.yml

```diff
@@ -0,0 +1,44 @@
+---
+manufacturer: Palo Alto
+model: PA-440
+slug: pa-440
+part_number: PA-440
+u_height: 1
+is_full_depth: false
+comments: '[Product Summary Specsheet - PAN-OS 10.1](https://www.paloaltonetworks.com/resources/datasheets/palo-alto-networks-product-summary-specsheet-pan-os-10-1)'
+power-ports:
+  - name: PS1
+    type: nema-1-15p
+  - name: PS2
+    type: nema-1-15p
+console-ports:
+  - name: console
+    type: rj-45
+  - name: usb-console
+    type: usb-mini-a
+  - name: usb1
+    type: usb-a
+  - name: usb2
+    type: usb-a
```

Comment:
This section is designated only for console ports, I believe USB1/2 are not actually console ports. If this is the case, please remove them